projects/addon-doc/components/example/example.component.ts

```diff
@@ -78,13 +84,27 @@ export class TuiDocExampleComponent {
             map(value => this.processContent(value)),
         );
 
+    protected readonly lazyComponent$ = this.lazyLoader$$.pipe(
+        switchMap(async lazyImport =>
```

Comment:
Maybe `switchAll` + `map` so we don't mix RxJS and async.then? If I remember correctly what switchAll does :)

Reply:
fixed